sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/models/_models.py

```diff
@@ -1098,7 +1098,7 @@ def _from_generated(cls, source):
         )
 
     def _to_generated_patch(self):
-        return _SqlSourceParameter(
+        return _SqlSourceParameterPatch(
```

Comment:
I think I saw that there is a `AzureDataExplorerDataFeedPatch` now so we don't have to use the SqlSourceParameterPatch.